packages/eslint-plugin-pf-codemods/lib/rules/refactor-breakpointmods.js

```diff
@@ -0,0 +1,129 @@
+const { getPackageImports } = require('../helpers');
+const componentsToUpdate = [
+  'Flex',
+  'FlexItem',
+  'PageSection',
+  'ToolbarItem',
+  'ToolbarToggleGroup'
+];
+const camelCase = str => str.replace(/-([a-z])/g, groups => groups[1].toUpperCase());
+const breakpointPropNames = {
+  visibility: ['hidden', 'visible'],
+  widths: ['width_25', 'width_33', 'width_50', 'width_66', 'width_75', 'width_100'],
+  padding: ['padding', 'noPadding'],
+  inset: ['insetNone', 'insetSm', 'insetMd', 'insetLg', 'insetXl', 'inset2xl'],
+  spacer: ['spacerNone', 'spacerXs', 'spacerSm', 'spacerMd', 'spacerLg', 'spacerXl', 'spacer2xl', 'spacer3xl', 'spacer4xl'],
+  spaceItems: ['spaceItemsNone', 'spaceItemsXs', 'spaceItemsSm', 'spaceItemsMd', 'spaceItemsLg', 'spaceItemsXl', 'spaceItems2xl', 'spaceItems3xl', 'spaceItems4xl'],
+  show: ['show'],
+  grow: ['grow'],
+  shrink: ['shrink'],
+  flex: ['flexDefault', 'flexNone', 'flex_1', 'flex_2', 'flex_3', 'flex_4'],
+  direction: ['column', 'columnReverse', 'row', 'rowReverse'],
+  alignItems: ['alignItemsFlexStart', 'alignItemsFlexEnd', 'alignItemsCenter', 'alignItemsStretch', 'alignItemsBaseline'],
+  alignContent: ['alignContentFlexStart', 'alignContentFlexEnd', 'alignContentCenter', 'alignContentStretch', 'alignContentSpaceBetween', 'alignContentSpaceAround'],
+  alignSelf: ['alignSelfFlexStart', 'alignSelfFlexEnd', 'alignSelfCenter', 'alignSelfStretch', 'alignSelfBaseline'],
+  align: ['alignLeft', 'alignRight'],
+  justifyContent: ['justifyContentFlexStart', 'justifyContentFlexEnd', 'justifyContentCenter', 'justifyContentSpaceBetween', 'justifyContentSpaceAround', 'justifyContentSpaceEvenly'],
+  display: ['inlineFlex'],
+  fullWidth: ['fullWidth'],
+  flexWrap: ['wrap', 'wrapReverse', 'nowrap']
+};
+const errorMessages = {
+  Flex: 'Removed breakpointMods prop from Flex in favor of spacer, spaceItems, grow, shrink, flex, direction, alignItems, alignContent, alignSelf, align, justifyContent, display, fullWidth and flexWrap',
+  FlexItem: 'Removed breakpointMods prop from FlexItem in favor of spacer, grow, shrink, flex, alignSelf, align, and fullWidth',
+  ToolbarItem: 'Removed breakpointMods prop from ToolbarItem in favor of visiblity, alignment, and spacer',
```

Comment:
Need PageSection in here. Sorry for my incorrect note!

Reply:
No update needed - PageSection error message is hardcoded inline as there are no dynamic use cases when reaching this error